packages/Avatar/src/__snapshots__/Avatar.test.tsx.snap

```diff
@@ -4,6 +4,7 @@ exports[`Avatar It should render a snapshot 1`] = `
 <DocumentFragment>
   <div
     class="ids-avatar ids-avatar--medium"
+    data-private="true"
```

Comment:
is that from another PR?

Reply:
I didn't refresh the snapshot last time. I don't remember it complaining.. weird. so you can ignore this.